pet_code/config/floodmaps.conf

```diff
@@ -1,11 +1,21 @@
 [filter]
+# Type of filter to apply, current options:
+# Impacts (minimum number of E channels),
+# OneMod (minimum number of E channels and only one minimod in each 'side'),
+# NoNeg (minimum number of E channels and no negative energies)
+# nSM (min E channels, no negatives and a max number of supermodules)
 type = Impacts
-min_channels = 4,4
+# minimum number of E channels
+min_channels = 4
+# Select only the mM with highest E channel sum in each 'side'?
```

Comment:
Select only the mM with highest E channel sum in each coincidence pair

Reply:
I've gone with "Select only the mM with the highest E channel sum in each half of the coincidence pair"